dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -304,6 +314,10 @@ public Response deleteToken(@ApiParam(hidden = true) @Parameter(hidden = true, n
     @Path("/gitlab.com")
     @JsonView(TokenViews.User.class)
     @Operation(operationId = "addGitlabToken", description = "Add a new gitlab.com token.", security = @SecurityRequirement(name = OPENAPI_JWT_SECURITY_DEFINITION_NAME))
+    @ApiResponse(responseCode = HttpStatus.SC_OK + "", description = "Successfully added a new gitlab.com token", content = @Content(schema = @Schema(implementation = Token.class)))
+    @ApiResponse(responseCode = HttpStatus.SC_BAD_REQUEST + "", description = "Bad request")
```

Comment:
can do a string constant replace for these three descriptions, your IDE can probably do a replacement across the whole file